device_operation/include/device_gemm_xdl_c_shuffle.hpp

```diff
@@ -29,22 +29,23 @@ template <
     ck::index_t BlockSize,
     ck::index_t MPerBlock,
     ck::index_t NPerBlock,
-    ck::index_t K0PerBlock,
-    ck::index_t K1,
+    ck::index_t KPerBlock,
     ck::index_t MPerXDL,
     ck::index_t NPerXDL,
     ck::index_t MXdlPerWave,
     ck::index_t NXdlPerWave,
     typename ABlockTransferThreadClusterLengths_K0_M_K1,
     typename ABlockTransferThreadClusterArrangeOrder,
     typename ABlockTransferSrcAccessOrder,
+    ck::index_t ABlockTransferK1PerBlock,
     ck::index_t ABlockTransferSrcVectorDim,
     ck::index_t ABlockTransferSrcScalarPerVector,
     ck::index_t ABlockTransferDstScalarPerVector_K1,
     bool ABlockLdsAddExtraM,
     typename BBlockTransferThreadClusterLengths_K0_N_K1,
```

Comment:
ABlockTransferThreadClusterLengths_AK0_N_AK1
BBlockTransferThreadClusterLengths_BK0_N_BK1

Reply:
Does renaming also applies to class methods, grid descriptor class name and their instances? Eg

GetABlockDescriptor_AK0PerBlock_MPerBlock_AK1()
GetBBlockDescriptor_BK0PerBlock_NPerBlock_BK1()
AGridDesc_AK0_M_AK1
BGridDesc_AK0_N_AK1
a_block_desc_ak0_m_ak1
b_block_desc_bk0_n_bk1